pkg/operator-kit/client.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
```suggestion
Copyright 2019 The Rook Authors. All rights reserved.
```

Reply:
This package will not be merged, I've just pulled it in so I can build with kube 1.14.1 I'll rebase it out before merging.